client/lib/preferences-helper/string-preference.tsx

```diff
@@ -45,18 +47,18 @@ const StringPreference: FunctionComponent< Props > = ( { name, value } ) => {
 			{ value !== localValue && (
 				<>
 					<button
-						className="preferences-helper__save-pref-button"
+						className="preferences-helper__preference-save"
 						onClick={ savePreferenceChange }
 						disabled={ value === localValue }
 					>
-						{ 'save' }
-					</button>{ ' ' }
+						{ translate( 'save' ) }
+					</button>
 					<button
-						className="preferences-helper__reset-pref-button"
+						className="preferences-helper__preference-reset"
 						onClick={ resetPreferenceChange }
 						disabled={ value === localValue }
 					>
-						{ 'reset' }
+						{ translate( 'reset' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **10** times:
`translate( 'Reset ' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/unnecessary-renders-in-prefs-helper)